Modules/_collectionsmodule.c

```diff
@@ -2127,6 +2130,42 @@ defdict_repr(defdictobject *dd)
     return result;
 }
 
+static PyObject*
+defdict_or(PyObject* left, PyObject* right)
+{
+    int lhs = PyObject_IsInstance(left, (PyObject*)&defdict_type);
```

Comment:
Sorry about harping on names! To me, `lhs` is a noun, but this variable is a condition -- the meaning of the condition seems to be the negation of "should the arguments be swapped". If it weren't for the negation I'd name it "swap" -- but given that, maybe "left_is_self"?